app/controllers/admin/dashboard_controller.rb

```diff
@@ -1,4 +1,6 @@
 class Admin::DashboardController < ApplicationController
   def index
+    @uncontacted_companies = Company.where(status: "uncontacted")
+    @contacted_companies = Company.where(status: "contacted")
```

Comment:
Sweet, I think it'll be good to be able to quickly have access to companies that are contacted or un-contacted. Is there a way to move the scopes into the model? Then if we have companies = Company.all we can call companies.contacted or companies.uncontacted? Trying to remember how that works.